examples/cloudrun-implicit-serverless-neg/variables.tf

```diff
@@ -0,0 +1,40 @@
+/**
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+variable "project_id" {
+  type = string
+}
+
+variable "region" {
+  description = "Location for load balancer and Cloud Run resources"
+  default     = "us-central1"
+}
+
+variable "ssl" {
+  description = "Run load balancer on HTTPS and provision managed certificate with provided `domain`."
+  type        = bool
+  default     = true
+}
+
+variable "domain" {
+  description = "Domain name to run the load balancer on. Used if `ssl` is `true`."
+  type        = string
+}
+
+variable "lb_name" {
+  description = "Name for load balancer and associated resources"
+  default     = "tf-cr-lb-1"
+}
```

Comment:
I know other examples follow the older but now we prefer to keep examples vars as lean as possible. So lets hard code all these in the examples with the values you are passing via fixture or defaulting to here.

Reply:
Done!